src/Appwrite/Platform/Tasks/CalcTierStats.php

```diff
@@ -356,4 +201,209 @@ public function action(Group $pools, Cache $cache, Database $dbForConsole, Regis
+            ]);
+        } catch (\Throwable) {
+            $stats['Domains'] = 0;
+        }
+
+        try {
+            /** Get Api keys */
+            $stats['Api keys'] = $dbForConsole->count('keys', [
+                Query::equal('projectInternalId', [$project->getInternalId()]),
+            ]);
+        } catch (\Throwable) {
+            $stats['Api keys'] = 0;
+        }
+
+        try {
+            /** Get Webhooks */
+            $stats['Webhooks'] = $dbForConsole->count('webhooks', [
+                Query::equal('projectInternalId', [$project->getInternalId()]),
+            ]);
+        } catch (\Throwable) {
+            $stats['Webhooks'] = 0;
+        }
+
+        try {
+            /** Get Platforms */
+            $stats['Platforms'] = $dbForConsole->count('platforms', [
+                Query::equal('projectInternalId', [$project->getInternalId()]),
+            ]);
+        } catch (\Throwable) {
+            $stats['Platforms'] = 0;
+        }
+
+            /** Get Files & Buckets */
+            $filesCount = 0;
+            $filesSum = 0;
+            $maxFileSize = 0;
+            $counter = 0;
+        try {
+            $buckets = $dbForProject->find('buckets', []);
+            foreach ($buckets as $bucket) {
+                $file = $dbForProject->findOne('bucket_' . $bucket->getInternalId(), [Query::orderDesc('sizeOriginal'),]);
+                if (empty($file)) {
+                    continue;
+                }
+                $filesSum   += $dbForProject->sum('bucket_' . $bucket->getInternalId(), 'sizeOriginal', []);
+                $filesCount += $dbForProject->count('bucket_' . $bucket->getInternalId(), []);
+                if ($file->getAttribute('sizeOriginal') > $maxFileSize) {
+                    $maxFileSize = $file->getAttribute('sizeOriginal');
+                }
+                $counter++;
+            }
+        } catch (\Throwable) {
+            ;
+        }
+            $stats['Buckets'] = $counter;
+            $stats['Files'] = $filesCount;
+            $stats['Storage (bytes)'] = $filesSum;
+            $stats['Max File Size (bytes)'] = $maxFileSize;
+
+
+        try {
+            /** Get Total Functions */
+            $stats['Databases'] = $dbForProject->count('databases', []);
+        } catch (\Throwable) {
+            $stats['Databases'] = 0;
+        }
+
+            /** Get Total Functions */
+        try {
+            $stats['Functions'] = $dbForProject->count('functions', []);
+        } catch (\Throwable) {
+            $stats['Functions'] = 0;
+        }
+
+            /** Get Total Deployments */
+        try {
+            $stats['Deployments'] = $dbForProject->count('deployments', []);
+        } catch (\Throwable) {
+            $stats['Deployments'] = 0;
+        }
+
+            /** Get Total Executions */
+        try {
+            $stats['Executions'] = $dbForProject->count('executions', []);
```

Comment:
please double check if the attribute names are the same as the ones we declared in the array above. 
If its incorrect, then the CSV will have missing info

Reply:
checked